pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -251,6 +251,7 @@ private static long getRandomInitialDelayInSeconds() {
   private static final String DEFAULT_DIM_TABLE_MAX_SIZE = "200M";
 
   private static final String DEFAULT_PINOT_FS_FACTORY_CLASS_LOCAL = LocalPinotFS.class.getName();
+  public static final String DISABLE_GROOVY = "controller.disable.groovy";
```

Comment:
Let's make it explicit that this disables groovy for ingestion, e.g. `controller.disable.ingestion.groovy`

Reply:
Shall we then change the broker property to `pinot.broker.disable.query.groovy`?

Reply:
Sure, we can make it explicit